R/PTXQC.R

```diff
@@ -26,6 +26,8 @@
 #' 
 #' @import data.table
 #' @import ggplot2
+#' @import stringr
```

Comment:
new packages should also be added to
`DESCRIPTION`, but only if they are really needed.

When calling a function from the package, prefix it with the package name,e.g. `stringr::someFunction(...)`. This lets you use the package without calling`library(stringr)` before.